faculty/clients/base.py

```diff
@@ -308,6 +331,40 @@ def _delete(self, endpoint, schema, **kwargs):
         response = self._delete_raw(endpoint, **kwargs)
         return _deserialise_response(schema, response)
 
+    @contextmanager
+    def _stream(self, endpoint):
+        """Stream from a SSE endpoint.
+        Usage
```

Comment:
Add empty line above "Usage"? (I think this is required or recommended by reStructuredText).